src/data/list/basic.lean

```diff
@@ -1390,6 +1390,22 @@ begin
     cases x, simpa using hxy, simp at hxy, simp [y_ih hxy.2, h hxy.1] }
 end
 
+/--
+A single `list.map` of a composition of functions is equal to
+composing a `list.map` with another `list.map`, fully applied.
+This is the reverse direction of `list.map_map`.
+-/
+lemma comp_map (l : list α) (g : α → β) (h : β → γ)  :
```

Comment:
```suggestion
lemma comp_map (h : β → γ) (g : α → β) (l : list α) :
```
With lemmas like this the list is the last argument. I personally put the functions in the order they appear in the statement, but that is not done consistently (but it is done in `list.map_map`).